docs/user/using-the-capture-app.md

```diff
@@ -1345,3 +1345,42 @@ The attribute option combo selector will be displayed when you are adding or cha
 Example from new Tracker event:
 
 ![](resources/images/attribute-option-combo-tracker.png)
+
+# Related stages and linked events for Tracker programs
+You can connect any two program stages via a relationship. To enable this feature, you can set up two program stages in the same program and link them with a stage-to-stage relationship.
```

Comment:
I would say event-event relationship as that is what it is labeled as in Maintenance etc and what we say for relationships (TE-event, TE-TE, event-event), just for clarity... Also, should we add a screenshot from that screen in Maintenance? (Always a bit unsure of how much to add of configuration screens)